paddle/fluid/platform/mkldnn_helper.h

```diff
@@ -42,6 +42,7 @@ inline mkldnn::memory::desc MKLDNNMemDesc(const std::vector<int>& dims,
 }
 
 inline bool CanMKLDNNBeUsed(const framework::ExecutionContext& ctx) {
+  if (!ctx.HasAttr("use_mkldnn")) return false;
```

Comment:
What's the purpose of line 45? Why don't previous conv2d #8451 and pool #8879 PR need this line?

Reply:
Done. I removed all changes. The idea was to hide the errors of testing when the framework was not builded with the GPU support (WITH_GPU=OFF)